internal/distro/rhel9/imagetype.go

```diff
@@ -41,6 +41,11 @@ const (
 
 	// blueprint package set name
 	blueprintPkgsKey = "blueprint"
+
+	// hardcode the datastream since this is provided by the scap-security-guide rpm
+	// and is not expected to change.
+	centosDatastream = "/usr/share/xml/scap/ssg/content/ssg-cs9-ds.xml"
+	rhelDatastream   = "/usr/share/xml/scap/ssg/content/ssg-rhel9-ds.xml"
```

Comment:
again, I would like to see these names prefixed with `oscap` to make it obvious what they are for...